timing_bench_test.go

```diff
@@ -46,7 +47,9 @@ func BenchmarkChannel(b *testing.B) {
 	b.ResetTimer()
 	for i := 0; i < concurrency; i++ {
 		go func() {
-			ch <- &timing{host: "myhost", dur: dur}
+			for j := 0; j < int(math.Ceil(float64(b.N)/concurrency)); j++ {
```

Comment:
Oh great improvement @oxtoacart on the benchmark thanks. I went ahead and made the channel buffered at 100, which I agree seems better. That whole exercise was super useful actually, so glad we went down that path. I feel like for really large concurrent applications that ability to offload less critical things to other go routines is likely pretty vital.